tests/manage/test_pvc_deletion_during_io.py

```diff
@@ -52,6 +52,13 @@ def test_run_io_and_delete_pvc(self):
             if "NotFound" in str(ex):
                 pass
 
+
+@pytest.mark.usefixtures(delete_pod.__name__)
+class TestRunningIO(TestDeletePVCWhileRunningIO):
```

Comment:
I don't think we should inherit from the test class.
Let's create a base class which the basic fixtures and inherit from it for both TCs

Reply:
I agree.
Will fix it